products/scripts/filler.mjs

```diff
@@ -0,0 +1,44 @@
+import AWS from "aws-sdk";
+import { v4 as uuid } from "uuid";
+import { getProductMocks } from "../mocks/products.mjs";
+
+const fillTable = async () => {
+  console.log(process.env.DB_STOCK,process.env.DB_PRODUCTS);
+
+  const mocks = await getProductMocks();
+  const dynamo = new AWS.DynamoDB.DocumentClient({region: 'us-east-1'});
+
+  mocks.forEach(item => {
+    try {
+      const itemId = uuid();
+
+      dynamo.put({
```

Comment:
There is `BatchWriteItem` method to put data in batches into dynamoDB, so you don't have to go one by one.